src/mixins/dom.js

```diff
@@ -0,0 +1,34 @@
+// DomMixin
+//  ---------
+
+import Backbone from 'backbone';
+
+export default {
+  createBuffer() {
+    return document.createDocumentFragment();
+  },
+
+  appendChildren(el, children) {
+    Backbone.$(el).append(children);
+  },
+
+  beforeEl(el, sibling) {
+    Backbone.$(el).before(sibling);
+  },
+
+  replaceEl(newEl, oldEl) {
+    Backbone.$(oldEl).replaceWith(newEl);
+  },
+
+  detachContents(el) {
+    Backbone.$(el).contents().detach();
+  },
+
+  attachHTML(el, html) {
+    Backbone.$(el).html(html);
+  },
+
+  removeEl(el) {
+    Backbone.$(el).remove();
```

Comment:
I get that but I am looking at it in terms of removing other elements other than `view.$el`

I can make a `removeEl` for `this.$el.remove();` and another one for other DOM like `removeDOM(dom)` if you like.